backend/src/user/user.controller.ts

```diff
@@ -66,8 +70,11 @@ export class UserController {
   @ApiOperation({ summary: "Edit data of specified user" })
   @ApiOkResponse({ description: "The resource was updated successfully" })
   @ApiNotFoundResponse({ description: "Resource not found" })
-  @ApiForbiddenResponse({ description: "Unauthorized Request" })
+  @ApiUnauthorizedResponse({ description: "Unauthorized Request" })
+  @ApiForbiddenResponse({ description: "Unauthorized Request, client does not have access rights to the requested content" })
```

Comment:
what's the difference between `ApiUnauthorizedResponse` and `ApiForbiddenResponse`?

Reply:
`ApiUnauthorizedResponse` is when the client provides no credentials or invalid credentials. `ApiForbiddenResponse` is the status code to return when a client has valid credentials but not enough privileges to perform an action on a resource. I guess maybe I could elaborate more in the `description`.

Reply:
ah okay can just add it in the swagger description!